tick/inference/tests/coxreg_test.py

```diff
@@ -52,10 +53,29 @@ def test_CoxRegression_fit(self):
 
             'tv': np.array([0.03017556, 0.03714465, 0.0385349, -0.10169967,
                             0.15783755, 0.64860815, -0.00617636, -0.22235137,
-                            -1.07938977, -0.07181225])
+                            -1.07938977, -0.07181225]),
+
+            'binarsity': np.array(
+                [0.03794176, -0.04473702, 0.00339763, 0.00339763, -0.16493989,
+                 0.05497996, 0.05497996, 0.05497996, -0.08457476, -0.08457476,
+                 0.0294825, 0.13966702, 0.10251257, 0.02550264, -0.07207419,
+                 -0.05594102, -0.10018038, -0.10018038, 0.10018038, 0.10018038,
+                 -0.47859686, -0.06685181, -0.00850803, 0.55395669, 0.00556327,
+                 -0.00185442, -0.00185442, -0.00185442, 0.26010429, 0.09752455,
+                 -0.17881442, -0.17881442, 0.932516, 0.32095387, -0.49766315,
+                 -0.75580671, 0.0593833, -0.01433773, 0.01077109, -0.05581666])
         }
 
         for penalty in self.penalties:
+
+            if penalty == 'binarsity':
+                # binarize features
+                n_cuts = 3
+                binarizer = FeaturesBinarizer(n_cuts=n_cuts)
+                features = binarizer.fit_transform(features_)
```

Comment:
ok for `raw_features`. The problem of overriding `features` is that if the binarsity penalty would have to be the last tested one in the for loop